plugins-weekly.ci.jenkins.io.txt

```diff
@@ -33,7 +33,7 @@ credentials-binding:657.v2b_19db_7d6e6d
 customizable-header:69.v36de0f11b_d00
 dark-theme:416.v535839b_c4e88
 data-tables-api:1.13.8-2
-datadog:5.6.2
+datadog:6.0.0
```

Comment:
https://github.com/jenkinsci/datadog-plugin/blob/master/CHANGELOG.md#600--2024-01-31 => no visible breaking changes. Hopefully the performance boost might help!